pkgs/applications/audio/clementine/default.nix

```diff
@@ -50,12 +50,12 @@ let
   withCloud = config.clementine.cloud or true;
 in mkDerivation {
   pname = "clementine";
-  version = "unstable-2022-04-11";
+  version = "unstable-2023-10-02";
 
   src = fetchFromGitHub {
     owner = "clementine-player";
     repo = "Clementine";
-    rev = "250024e117fbe5fae7c62b9c8e655d66412a6ed7";
+    rev = "7b678f26e07faeeea7c3fc61d147c9693e29aa32";
```

Comment:
Why this commit?

Reply:
It is the latest commit unless I did something wrong. Just bumping to the newest possible, as the package is running a non-release version anyway and the current commit is pretty outdated

Reply:
latest commit is from last week: https://github.com/clementine-player/Clementine

Reply:
Updated to last commit